binding/go/porcupine_test.go

```diff
@@ -13,28 +13,61 @@ package porcupine
 import (
 	"encoding/binary"
 	"flag"
+	"fmt"
 	"io/ioutil"
 	"math"
 	"os"
 	"path/filepath"
 	"testing"
+	"reflect"
 )
 
-var pvTestAccessKey string
+var (
+	pvTestAccessKey string
```

Comment:
I know this is not part of this code change but I rather drop the `pv` here. You can use something like `testAccessKey` or even `accessKey`.